mods/flowers/init.lua

```diff
@@ -101,15 +160,15 @@ for _, m in pairs(mushrooms_datas) do
 		}
 	})
 
-	-- Register spores
+	-- Register mushroom spores
 
-	minetest.register_node("flowers:spores_" .. name, {
+	minetest.register_node("flowers:mushroom_spores_" .. name, {
 		description = string.sub(string.upper(name), 0, 1) ..
 			string.sub(name, 2) .. " Mushroom Spores",
```

Comment:
I actually tested and it worked fine. So whether you're lying and you should clean your glasses.